### PR TITLE
sevctl-verify: Return error to main, rather than exiting

### LINKED ISSUE #53: sevctl-verify should return an error to main instead of exiting when verification fails
All of the other code paths do this. The main function will terminate with a bad status code if it catches and emits the errors.

### LINKED ISSUE #54: sevctl-verify should not emit the icon legend when invoked with --quiet
I.e.,

```console
$ ./target/release/sevctl verify --quiet

 • = self signed, ⬑ = signs, •̷ = invalid self sign, ⬑̸ = invalid signs
 ```
 
 should just be:
 
 ```console
 $ ./target/release/sevctl verify --quiet
 $ echo $?
0
 ```

### PR DESCRIPTION
**Pull request by tylerfanelli**
Proposed fix for #53 that includes the omission of an error message if "verify --quiet" is enabled. This allows sevctl verify to be uniform with all other subcommands.

### CODE REVIEW

**Review by connorkuehl — CHANGES_REQUESTED** [review]
@tfanelli-rh Have you tried hoisting the `quiet` flag out to be a top-level argument to sevctl? If the other commands fail, they will propagate errors down to main and main will print them and there's currently no way for the operator to make those commands quiet as well. I think for that reason, we could decouple the `quiet` flag from the `verify` command and make it a top-level switch.

Also, please remove the "." from the end of your commit subject.

**Comment by tfanelli-rh** [comment]
Latest commit has also addressed #53 by checking if quiet is enabled before printing anything to the console.

**Comment by tfanelli-rh** [comment]
Latest commit has also addressed #53 by checking if quiet is enabled before printing anything to the console.

**Comment by tylerfanelli** [comment]
Latest commit has also addressed #54  by checking if quiet is enabled before printing anything to the console.

**Review by connorkuehl — CHANGES_REQUESTED** [review]
1. Please rebase and fix the merge conflicts.
2. Is it possible to avoid duplicating the quiet field across all variants of the enum? A quick look at the [StructOpt documentation for subcommands](https://docs.rs/structopt/0.3.21/structopt/#subcommands) suggests we might be able to turn `enum Sevctl` into a `struct Sevctl` and factor the variants out into their own enum. Then the new `struct Sevctl` could expose a top-level `quiet` flag perhaps. 
3. Thanks for taking care of the legend-printing issue! :slightly_smiling_face:

**Review by connorkuehl — CHANGES_REQUESTED** [review]
Overall, this looks good to me. Just two suggestions inline.

File: src/main.rs [hunk 1]
```diff
@@ -397,12 +402,19 @@ mod verify {
...
 
         if err as i32 == 0 {
             Ok(())
         } else {
-            exit(err as i32)
+            let certificate_error: std::io::Error = std::io::ErrorKind::InvalidData.into();
+            Err(error::Context::new(
+                "SEV/CA certificate verification failed.",
+                Box::new(certificate_error),
+            ))
```
[the remark this answers is not in the document]
Comment (tylerfanelli) [reply src/main.rs]: Yea, I see what you mean. I changed this in my latest commit.

**Review by connorkuehl — APPROVED** [review]
Thanks, @tfanelli-rh! Looks good to me.

**Pull request merged**